apps/passport-client/components/screens/LoginScreens/LoginScreen.tsx

```diff
@@ -126,6 +130,93 @@ export function LoginScreen(): JSX.Element {
     [dispatch, email]
   );
 
+  const [tryStorageAccess, setTryStorageAccess] = useState(false);
+  const [tryStorageLogin, setTryStorageLogin] = useState(false);
+
+  /**
+   * Assuming we're in Chrome and an iframe, and we've successfully loaded an
+   * encryption key from local storage, try to use it to log in.
+   */
+  const tryToLogin = useCallback(
+    async (encryptionKey: string) => {
+      // Try to download and decrypt the storage
+      const storageRequest = await requestDownloadAndDecryptStorage(
+        appConfig.zupassServer,
+        encryptionKey
+      );
+      if (storageRequest.success) {
```

Comment:
What happens if this fails?  There's no signaling of an error to the user, or returning of a value here, which is surprising.  Seems like this needs similar error handling, and maybe a loading spinner during download, similar to other pages which do this download and login.

Reply:
I've added an error message in the case where this fails.